### DISCUSSION TITLE
Feature Request: disabled prop for Select

### DISCUSSION THREAD

**Discussion by megacherry** [tamagui/tamagui#2203] [General]
### Current Behavior

Hi Nate,

can we add a disabled prop for the Select component?

Thanks :)

Andy

### Tamagui Version

```markdown
1.89.0
```

### Platform (Web, iOS, Android)

```markdown
all of them
```

### Reproduction

```markdown
none
```

**Comment by cmpickle** [comment]
I also need this functionality. Was this turned into a discussion because you possibly don't want this implemented? I am willing to make a PR to add this functionality but don't want to go through the work if it won't get merged.

**Comment by megacherry** [comment]
@cmpickle: yes, it was converted, but i also don't know why.

@bidah/@TheEhsanSarshar: Can you say something about it?

**Comment by byteab** [comment]
adding `disabled` to the Select.Trigger should do the job ?

**Comment by megacherry** [comment]
@TheEhsanSarshar: works, thank you :)

**Comment by RadomirPerisic** [comment]
It doesn't work with `native` prop on Select component since Select.Trigger isn't rendered in this case
